aif360/metrics/mdss/ScoringFunctions/ScoringFunction.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+class ScoringFunction:
+
+    def __init__(self, **kwargs):
```

Comment:
Some comment on what this class is. Abstract class?

Reply:
Yes, an abstract class.